testcases/PowerNVDump.py

```diff
@@ -79,9 +79,10 @@ def setUp(self):
         self.pdbg = conf.args.pdbg
         self.basedir = conf.basedir
         self.c = self.cv_SYSTEM.console
-        #crash_content : a variable to point to right crash folder after core is
+        self.cv_SYSTEM.goto_state(OpSystemState.OS)
```

Comment:
Why are you adding system state to common class? Better call `setup_test` from SBECrash_MPIPL?

Reply:
I am seeing console login failures while running below os command.. tried calling `setup_test` but that would be called later.. which did not not solve the issue what I was seeing at this point..